train-26 tests/functional/fonts.js test failing

This is failing for me against https://accounts.stage.mozaws.net/ and https://latest.dev.lcip.org/ with FF 33 on osx and ubuntu.

```
Running: fonts
Test main - fonts - Does not use Fira for non-supported locale FAILED on firefox 33.1 on MAC:
AssertionError: expected false to be truthy
  at Function.assert.ok  <node_modules/intern-geezer/assert.js:525:10>
  at CompatCommand.<anonymous>  <tests/functional/fonts.js:53:20>
  at runCallback  <node_modules/intern-geezer/node_modules/leadfoot/Command.js:509:31>
  at CompatCommand.<anonymous>  <node_modules/intern-geezer/node_modules/leadfoot/Command.js:526:11>
  at <node_modules/intern-geezer/node_modules/leadfoot/node_modules/dojo/Promise.js:256:37>
  at runCallbacks  <node_modules/intern-geezer/node_modules/leadfoot/node_modules/dojo/Promise.js:13:18>
  at <node_modules/intern-geezer/node_modules/leadfoot/node_modules/dojo/Promise.js:204:17>
  at Object.run [as _onImmediate]  <node_modules/intern-geezer/node_modules/leadfoot/node_modules/dojo/Promise.js:143:29>
  at processImmediate [as _immediateCallback]  <timers.js:345:15>
  at new CompatCommand  <node_modules/intern-geezer/runner.js:212:14>
  at CompatCommand.Command.then  <node_modules/intern-geezer/node_modules/leadfoot/Command.js:525:10>
  at registerSuite.Does not use Fira for non-supported locale [as test]  <tests/functional/fonts.js:52:12>
  at declare.run  <node_modules/intern-geezer/lib/Test.js:166:19>
```

--- a/app/styles/system-font-main.scss
+++ b/app/styles/system-font-main.scss
@@ -4,7 +4,7 @@
   font: message-box;
 }
 
-@mixin header-font() {
+@mixin headerFont() {
   font: message-box;
 }
 

